src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,67 @@
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+
+interface Props{
+  person: Person,
+  people: Person[],
+}
+
+export const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const { slug } = useParams<{ slug: string }>();
+
+  const motherInList = people.find(
+    mother => person.motherName === mother.name,
+  );
+
+  const fatherInList = people.find(
+    father => person.fatherName === father.name,
+  );
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({
+        'has-background-warning': slug === person.slug,
+      })}
+    >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
+          className={cn({ 'has-text-danger': person.sex === 'f' })}
+        >
+          {person.name}
+        </Link>
+      </td>
+
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
+      <td>{person.died}</td>
+      <td>
+        {motherInList
+          ? (
+            <Link
+              to={`/people/${motherInList.slug}`}
+              className="has-text-danger"
+            >
+              {person.motherName}
+            </Link>
```

Comment:
this part should be named as `PersonLink`
![image](https://github.com/mate-academy/react_people-table-basics/assets/22824431/3aace841-9912-4f9b-850e-888303a5d412)